build.gradle

```diff
@@ -277,7 +279,7 @@ swaggerSources {
                 "--model-package", "bio.terra.model",
                 "-D", "interfaceOnly=true," +
                     "useTags=true," +
-                    "dateLibrary=java8"
+                    "dateLibrary=java11"
```

Comment:
The config options here mention `java8` is preferred for jdk 1.8+ (there is no `java11` option at the moment): https://openapi-generator.tech/docs/generators/java/

Reply:
This makes sense because the library was new in JDK 8 but hasn't been updated since. So java8 is really java8+